### PR TITLE
Updated index.html with my profiles

### PR DESCRIPTION
**Pull request by BalajiG2000**
I have added my github and linkedin profiles in index.html file.

### CODE REVIEW

**Comment by anku580** [comment]
Hi @BalajiG2000 , there are some conflicts, can you please resolve the conflict?

File: index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
     </div>
     <div class="d-flex flex-column">
       <ul>
-      <li><a href="https://github.com/anku580" target="blank">Anku</a></li>
+      <li><a href="https://github.com/BalajiG2000" target="blank">Balaji</a></li>
```
Comment (anku580) [inline index.html]: Instead of modifying my name, simply add yours in the new `<li> tag.

File: index.html [hunk 2]
```diff
@@ -45,10 +45,10 @@
     </div>
     <div class="d-flex flex-column">
       <ul>
-      <li><a href="https://www.linkedin.com/in/anku580/" target="blank">Anku</a></li>
+      <li><a href="https://www.linkedin.com/in/balaji-g-3bb1a1182/" target="blank">Anku</a></li>
```
Comment (anku580) [inline index.html]: Instead of modifying my name, simply add yours in the new `<li> tag.

**Review by anku580 — CHANGES_REQUESTED** [review]
Please update your PR as per the comments provided. 

Thank you for your valuable contribution.

**Comment by BalajiG2000** [comment]
@anku580  Can you please elaborate? , Should I just add my name and profile , instead of replacing yours?

**Comment by anku580** [comment]
@BalajiG2000 yes, insead of removing my name, you can add your name in a new `<li>` tag.

**Comment by anku580** [comment]
There are some conflicts with this PR, can you please resolve it and update the PR?

File: index.html [hunk 3]
```diff
@@ -1,56 +1,56 @@
...
+    <div class="row mt-3">
+      <div class="col-md-12">
+        <img src="./git.png" height="20px">
+        <span class="git-header">Github Detail</span>
+      </div>
+    </div>
+    <div class="d-flex flex-column">
+      <ul>
+      <li><a href="https://github.com/anku580" target="blank">Anku</a></li>
+      <li><a href="https://github.com/BalajiG2000" target="blank">Balaji</a></li>
```
Comment (anku580) [inline index.html:39]: Please don't replace anyone else username as it would delete it. You can add it one after the other.

File: index.html [hunk 4]
```diff
@@ -1,56 +1,56 @@
...
+    <div class="row mt-3">
+      <div class="col-md-12">
+        <img src="./linkedin.jpg" height="20px">
+        <span class="linkedin-header">Linkedin Detail</span>
+      </div>
+    </div>
+    <div class="d-flex flex-column">
+      <ul>
+        <li><a href="https://www.linkedin.com/in/anku580/" target="blank">Anku</a></li>
+       <li><a href="https://www.linkedin.com/in/balaji-g-3bb1a1182/" target="blank">Anku</a></li>
```
Comment (anku580) [inline index.html]: Please don't replace anyone else username as it would delete it. You can add it one after the other.

**Review by anku580 — CHANGES_REQUESTED** [review]
Please don't replace anyone username.

File: index.html [hunk 5]
```diff
@@ -1,57 +1,116 @@
...
+      <ul>
+        <li><a href="https://www.linkedin.com/in/anku580/" target="blank">Anku</a></li>
+       
+    </ul>
+    </div>
+  </div>
+  </div>
+</body>
+</html>
+<!DOCTYPE html>
```
Comment (anku580) [inline index.html:1]: Hi buddy, why you have added the duplicated code? You need to remove the duplicate code.

**Review by anku580 — CHANGES_REQUESTED** [review]
Please remove the redundant code.

**Comment by anku580** [comment]
@BalajiG2000 There are some merge conflicts coming. If you want to learn how to resolve merge conflicts, you can watch this [youtube video](https://youtu.be/Mrh_YmPqNU8) created by me.

**Pull request closed without merging**